ios/brave-ios/Sources/Brave/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -384,9 +384,21 @@ public class AutocompleteTextField: UITextField, UITextFieldDelegate {
   // Reset the cursor to the end of the text field.
   // This forces `caretRect(for position: UITextPosition)` to be called which will decide if we should show the cursor
   // This exists because ` caretRect(for position: UITextPosition)` is not called after we apply an autocompletion.
+  private func resetCursor() {
+    guard let text = self.text,
+      let endPosition = self.position(from: self.beginningOfDocument, offset: text.count)
```

Comment:
Why not just compare `self.selectedTextRange.start` to `self.endOfDocument`?

Reply:
Fixed.